astroships/api/math/vector2.nim

```diff
@@ -0,0 +1,3 @@
+type
```

Comment:
This is totally fine, just wanted to note that this is also avail in: [nico/vec](https://github.com/ftsf/nico/blob/master/nico/vec.nim)